Makefile

```diff
@@ -12,9 +12,25 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-.PHONY: gtctl
+ifeq (,$(shell go env GOBIN))
+GOBIN=$(shell go env GOPATH)/bin
+else
+GOBIN=$(shell go env GOBIN)
+endif
 
+.PHONY: gtctl
 LDFLAGS = $(shell ./hack/version.sh)
-
 gtctl:
 	@go build -ldflags '${LDFLAGS}' -o bin/gtctl ./cmd
+
+.PHONY: ginkgo
```

Comment:
I think it's no need to install `ginko` here. You can follow greptimedb-operator.